hw/ip_templates/otp_ctrl/dv/env/otp_ctrl_env_cov.sv.tpl

```diff
@@ -274,6 +288,112 @@ class otp_ctrl_env_cov extends cip_base_env_cov #(.CFG_T(otp_ctrl_env_cfg));
     dai_access_secret2: cross lc_creator_seed_sw_rw_en, dai_access_cmd;
   endgroup
 
+
+  covergroup zr_dai_cmd_cg with function sample(
+        otp_ctrl_part_pkg::part_idx_e part_idx, bit zeroizable, otp_part_addr_cov_e offset_addr);
+    coverpoint part_idx
+    {
+      ignore_bins _ignore_bins = {otp_ctrl_part_pkg::LifeCycleIdx,
+                                  otp_ctrl_part_pkg::DaiIdx,
+                                  otp_ctrl_part_pkg::LciIdx,
+                                  otp_ctrl_part_pkg::KdiIdx,
+                                  otp_ctrl_part_pkg::NumAgentsIdx};
+    }
+    coverpoint zeroizable;
+    coverpoint offset_addr;
+
+    zeroized_partition: cross part_idx, zeroizable, offset_addr
+    {
+      ignore_bins part_not_zeroizable =
+           binsof (zeroizable) intersect {1}
+        && binsof (part_idx) intersect {
+% for part in non_zeroizable_parts:
+          ${Name.to_camel_case(part["name"])}Idx${"" if loop.last else ","}
+% endfor
+        };
+
+      ignore_bins part_not_zeroizable_no_digest_addr =
+           binsof (zeroizable) intersect {0}
+        && binsof (part_idx) intersect {
+% for part in zr_unbuffered_parts_without_digest:
+          ${Name.to_camel_case(part["name"])}Idx${"" if loop.last else ","}
+% endfor
+        }
+        && binsof (offset_addr) intersect {OtpPartDigestAddr};
+
+      ignore_bins part_not_zeroizable_no_zero_addr =
+           binsof (zeroizable) intersect {0}
+        && binsof (part_idx) intersect {
+% for part in non_zeroizable_parts:
+          ${Name.to_camel_case(part["name"])}Idx${"" if loop.last else ","}
+% endfor
+        }
+        && binsof (offset_addr) intersect {OtpPartZeroAddr};
+
+      ignore_bins part_zeroizable =
+           binsof (zeroizable) intersect {0}
+        && binsof (part_idx) intersect {
+% for part in zeroizable_parts:
+          ${Name.to_camel_case(part["name"])}Idx${"" if loop.last else ","}
+% endfor
+        };
+    }
+  endgroup
+
+  covergroup zr_partition_read_cg with function sample(
```

Comment:
This covergroup applies to any partition, so the "zr_" prefix seems inappropriate. Could you remove it?